README.md

```diff
@@ -645,34 +653,54 @@ The purpose of equity compensation is threefold:
-  an [**IPO**](https://en.wikipedia.org/wiki/Initial_public_offering). A sale or IPO is often
-  called an **exit** or **liquidity event**. Sales, dissolutions, and bankruptcy are all called
-  **liquidations**.
-  - 🚧 Expansion and clarification of “becoming public”?
-
-Finally, note that one other way stock is valuable is that public and some private
-companies can pay [**dividends**](http://www.investopedia.com/terms/d/dividend.asp) (allocations
-of a company’s profit determined by the company’s board) to shareholders, but this is rare
-among startups and companies focused on rapid growth.
-
-- 🚧 Explain more why this is.
-  Define dividends better.
+With public companies, you can usually sell any stock you won outright, so the answer is
+relatively easy:
+You look up the current market value of the stock.
+
+🚧 Mention lockup periods etc.
+
+❗️ With private companies, it’s hard to know the “real value” of stock and stock options.
+Because the value of private company stock is not determined by public trades,
+shareholders can only make educated guesses about the likely future value, at a time when
+they will be able to buy or sell the stock.
+It’s always worth remembering private company stock is simply a legal agreement that
+entitles you to something of highly uncertain value, and could well be worthless in the
+future, or highly valuable, depending on the fate of the company.
+
+☝️ We’ll discuss the notion of [fair market value](#409a-valuations) later, but even if a
+company assigns a value to the stock for tax and accounting purposes, it doesn’t mean you
+can expect to sell it for that value!
+
+### IPOs, sales, and liquidity
+
+🄳 A private company becomes a public company (or “**goes public**”) in a process called an
+[**initial public offering (or IPO)**](https://en.wikipedia.org/wiki/Initial_public_offering).
+Historically, only private companies with a strong track record of years of growth and
+profitability have considered themselves ready to take on this significant step, which
+includes a lot of regulatory costs
+[in exchange for the benefits](https://www.investopedia.com/university/ipo/ipo.asp) of
+significant capital to grow and expand.
+After a company goes public, investors and the general public can buy stock, and existing
+shareholders can sell their stock far more easily than when the company was private.
+
+🚧 What are the restrictions and regulations on selling stock that affect employees at
+IPO? Define lockup period.
+
+🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
+companies, it is often hard to sell stock until the company is sold or IPOs, so there is
+little or no liquidity for shareholders until those events occur.
+
+🄳 A sale or IPO is often called an **exit** or **liquidity event**. Sales, dissolutions, and
+bankruptcy are all called **liquidations**.
+
+🄳 A [**dividend**](http://www.investopedia.com/terms/d/dividend.asp) is a distribution of a
```

Comment:
Brief explanation of why a company would choose to pay dividends if shareholders(in public companies) are able to make money from their shares? A loyalty program?

Reply:
(Not clear to me why dividends are in this section in any case)